src/Security/Authentication/OpenIdConnect/src/OpenIdConnectOptions.cs

```diff
@@ -255,6 +261,14 @@ public new OpenIdConnectEvents Events
     /// </summary>
     public ISecurityTokenValidator SecurityTokenValidator { get; set; }
 
+    /// <summary>
+    /// Gets or sets the <see cref="TokenHandler"/> used to validate identity tokens.
+    /// <para>
+    /// This will be used instead of <see cref="SecurityTokenValidator"/> if <see cref="UseTokenHandler"/> is <see langword="true"/>
+    /// </para>
+    /// </summary>
+    public TokenHandler TokenHandler { get; set; }
```

Comment:
The other PR returns `public IList<TokenHandler> TokenHandlers { get; private set; }` should everything be aligned or does that matter?

Reply:
resolved